### PR TITLE
Update exchange.json

### PR DESCRIPTION
**Pull request by EOEboh**
Added MTN Group and Globacom Limited

### CODE REVIEW

**Comment by EOEboh** [comment]
@LhaosuBaby any reason why this PR hasn't been merged yet?

File: data/operators/telecom/exchange.json [hunk 1]
```diff
@@ -180,6 +180,19 @@
...
     },
+    {
+      "displayName": "Glo",
+      "locationSet": {
+        "include": ["ng", "bj"],
+      },
+      "tags": {
+        "name": "Glo",
+        "operator": "Globacom",
+        "operator:wikidata": "Q66365656",
```
Comment (LaoshuBaby) [inline data/operators/telecom/exchange.json:191]: Those Wikidata and Wikipedia don't match

[Q66365656 (Glo (company))](https://www.wikidata.org/wiki/Q66365656) don't relevent to any Wikipedia pages, and I can't find a page named with "Globacom Limited" in English Wikipedia.

I only can found a [Glo (company)](https://en.wikipedia.org/wiki/Glo_(company)), and its related Wikidata is [Q5570033 (Globacom)](https://www.wikidata.org/wiki/Q5570033)

File: data/operators/telecom/exchange.json [hunk 2]
```diff
@@ -242,6 +255,19 @@
         "telecom": "exchange"
       }
     },
+    {
+      "displayName": "MTN Group",
+      "locationSet": {
+        "include": ["za", "ng", "gh", "bj", "bw", "cm", "af", "cg", "gn", "gw", "ir", "ci", "lr", "rw", "ss", "sd", "sy", "sz", "ug", "ye", "zm" ]
```
Comment (LaoshuBaby) [inline data/operators/telecom/exchange.json:261]: So many countries, I'd prefer think this company is global and use "001" instead.

(MTN's wikidata and wikipedia is correct)

**Review by LaoshuBaby — CHANGES_REQUESTED** [review]

File: data/operators/telecom/exchange.json (hunk 1 above)
Comment (EOEboh) [reply data/operators/telecom/exchange.json:191]: Sure. Noted

**Review by arch0345 — APPROVED** [review]
Thanks @EOEboh

**Pull request merged**